### DISCUSSION TITLE
Debug SSH kitten issues

### DISCUSSION THREAD

**Discussion by memco** [kovidgoyal/kitty#6102] [Q&A] [answered]
When I SSH to a server without using the kitten it works, but when I try to use the kitten I get:

```console
/usr/local/bin/fish: No such file or directory
```

I am not sure whether that's local or remote nor what options control it. On my local machine I use fish, but it has a different path and on the remote I do not have specific configs for the shell and the default is bash.

I have this in `~/.config/kitty/ssh.conf`
```
shell_integration disabled
```

I also tried adding `interpreter bash`,  but neither of those settings made a difference.

Adding -vvv does not tell me more about that error.

How can I figure out where that message is coming from and how to fix it?

**Comment by kovidgoyal** [comment, marked as answer]
The ssh kitten does not run shells locally, only remotely. I suggest you
set shell in ssh.conf to the absolute path of whatever shell is
installed on your remote machine.

**Reply by memco** [reply]
Thanks! I tried setting `login_shell`, but had the same issue.  I ended up uninstalling and re-installing Kitty and that was enough to fix the issue. Not sure what was going on.
